samples/bluetooth/central_iso/src/main.c

```diff
@@ -133,20 +144,17 @@ static void iso_connected(struct bt_iso_chan *chan)
 {
 	printk("ISO Channel %p connected\n", chan);
 
-	seq_num = 0U;
-
-	/* Start send timer */
-	k_work_schedule(&iso_send_work, K_MSEC(0));
+	iso_send_sdu(1U, -1);
 }
 
 static void iso_disconnected(struct bt_iso_chan *chan, uint8_t reason)
 {
 	printk("ISO Channel %p disconnected (reason 0x%02x)\n", chan, reason);
-	k_work_cancel_delayable(&iso_send_work);
 }
 
 static struct bt_iso_chan_ops iso_ops = {
 	.connected	= iso_connected,
+	.sent = iso_sent,
 	.disconnected	= iso_disconnected,
```

Comment:
```suggestion
	.connected	= iso_connected,
	.disconnected	= iso_disconnected,
	.sent           = iso_sent,
```

or

```suggestion
	.connected = iso_connected,
	.disconnected = iso_disconnected,
	.sent = iso_sent,
```

Personal opinion, but it kind of stood out with `sent` in the middle of the connected and disconnected callbacks, and also not using the same indentation scheme :D

Reply:
The sent callback is no longer needed, so I returned to the previous code. This should therefore be obsolete.